apps/web/src/server/applications/search-results/applications-search.controller.js

```diff
@@ -26,7 +26,7 @@ export async function searchApplications(req, response) {
 		});
 	}
 
-	const pageSize = 50;
+	const pageSize = 1;
```

Comment:
why this change to the search page?

Reply:
will put back - was testing how the pagination works